resources/lang/ru/settings.php

```diff
@@ -42,21 +42,21 @@
     'app_disable_comments_desc' => 'Отключение комментов на всех страницах. Существующие комментарии отображаться не будут.',
 
     // Color settings
-    'content_colors' => 'Content Colors',
-    'content_colors_desc' => 'Sets colors for all elements in the page organisation hierarchy. Choosing colors with a similar brightness to the default colors is recommended for readability.',
-    'bookshelf_color' => 'Shelf Color',
-    'book_color' => 'Book Color',
-    'chapter_color' => 'Chapter Color',
-    'page_color' => 'Page Color',
-    'page_draft_color' => 'Page Draft Color',
+    'content_colors' => 'Цвета контента',
+    'content_colors_desc' => 'Настройка цветов элементов иерархии страницы. Для комфортного чтения рекомендуем выбирать цвета близкие по яркости к цветам по умолчанию.',
+    'bookshelf_color' => 'Цвет полки',
+    'book_color' => 'Цвет книги',
+    'chapter_color' => 'Цвет главы',
+    'page_color' => 'Цвет страницы',
+    'page_draft_color' => 'Цвет страницы черновика',
 
     // Registration Settings
     'reg_settings' => 'Настройки регистрации',
     'reg_enable' => 'Разрешить регистрацию',
     'reg_enable_toggle' => 'Разрешить регистрацию',
     'reg_enable_desc' => 'Если регистрация разрешена, пользователь сможет зарегистрироваться в системе самостоятельно. При регистрации назначается роль пользователя по умолчанию',
     'reg_default_role' => 'Роль пользователя по умолчанию после регистрации',
-    'reg_enable_external_warning' => 'The option above is ignored while external LDAP or SAML authentication is active. User accounts for non-existing members will be auto-created if authentication, against the external system in use, is successful.',
+    'reg_enable_external_warning' => 'Если используется LDAP или SAMP аутентификация, то параметр выше игнорируется. Аккаунты для несуществующих пользователей будут автоматически созданы если аутентификация на внешней системе будет успешна.',
```

Comment:
```suggestion
    'reg_enable_external_warning' => 'Если используется LDAP или SAMP аутентификация, то параметр выше игнорируется. Аккаунты для несуществующих пользователей будут автоматически созданы если аутентификация на внешней системе будет успешна.',
```

Typo - SAMP should be SAML?

Reply:
ok